scalafix-testkit/src/main/scala/scalafix/testkit/DiffTest.scala

```diff
@@ -25,34 +25,39 @@ object DiffTest {
   private def stripPrefix(str: String) = PrefixRegex.replaceFirstIn(str, "")
 
   def fromSemanticdbIndex(index: SemanticdbIndex): Seq[DiffTest] =
-    index.documents.map { document =>
+    index.documents.flatMap { document =>
       val input @ Input.VirtualFile(label, code) = document.input
-      val relpath = RelativePath(label)
-      val config: () => (Rule, ScalafixConfig) = { () =>
-        input.tokenize.get
-          .collectFirst {
-            case Token.Comment(comment) =>
-              val decoder =
-                ScalafixReflect.fromLazySemanticdbIndex(
-                  LazySemanticdbIndex(_ => Some(index)))
-              ScalafixConfig
-                .fromInput(
-                  Input.VirtualFile(label, stripPrefix(comment)),
-                  LazySemanticdbIndex(_ => Some(index)))(decoder)
-                .get
-          }
-          .getOrElse(throw new TestFailedException(
-            s"Missing scalafix configuration inside comment at top of file $relpath",
-            0))
+      if (!label.startsWith("scalafix-tests")) {
```

Comment:
This seems to be required by the introduction of `dependencies` to the classpath, if we remove that we can probably revert this guard.